src/components/DomEvents.js

```diff
@@ -126,6 +127,12 @@ function DomEvents() {
     setEventCount(0);
   };
 
+  const getTextToCopy = () => {
+    return buffer.current
+      .map((log) => `${log.target.toString()} - ${log.event.EventType}`)
+      .join('\n');
+  };
```

Comment:
```suggestion
  const getTextToCopy = () =>
    buffer.current
      .map((log) => `${log.target.toString()} - ${log.event.EventType}`)
      .join('\n');
```

Reply:
If we want to follow a pattern strictly, it would be better if we add the [rule](https://eslint.org/docs/rules/arrow-body-style) to eslintrc/prettierrc

Reply:
I agree. Let's leave it here the way it is. 

If someone can open a PR to add `arrow-body-style` to the eslint linter config, that would be most welcome.